src/usecases/retrieveWardById.js

```diff
@@ -3,14 +3,35 @@ const retrieveWardById = ({ getDb }) => async (wardId, trustId) => {
   console.log("Retrieving ward for  ", wardId);
   try {
     const ward = await db.oneOrNone(
-      "SELECT * FROM wards WHERE id = $1 AND trust_id = $2 LIMIT 1",
+      `SELECT
+        wards.id as ward_id,
+        wards.name as ward_name,
+        (
+          SELECT
+            name
+          from
+            hospitals
+          where
+            id = wards.hospital_id
```

Comment:
I believe we need to update this query to include the trust ID when retrieving the hospital (same for https://github.com/madetech/nhs-virtual-visit/pull/294/files#diff-d8c3299b5e0b2cd1d5c6fc7d55081ad5R9-R15)

Reply:
I've pushed what I think you mean :)